python/runfiles/runfiles.py

```diff
@@ -247,6 +247,15 @@ def CurrentRepository(self, frame: int = 1) -> str:
             raise ValueError("failed to determine caller's file path") from exc
         caller_runfiles_path = os.path.relpath(caller_path, self._python_runfiles_root)
         if caller_runfiles_path.startswith(".." + os.path.sep):
+            # With Python 3.10 and earlier, sys.path contains the directory
```

Comment:
nit: mention the PYTHONSAFEPATH envvar. Python 3.11+ is a necessary but not sufficient condition to trigger this situation.